packages/logger/src/formatters/test/ConsoleFormatter.test.ts

```diff
@@ -0,0 +1,114 @@
+import {ConsoleFormatter} from '../ConsoleFormatter';
+import {LogLevel} from '../..';
+
+describe('ConsoleFormatter', () => {
+  it('logs Critical entries to console.error', () => {
+    const formatter = new ConsoleFormatter();
+    const errorSpy = jest.spyOn(console, 'error');
+    const errorMsg = 'foo';
+
+    formatter.next({
+      level: LogLevel.Critical,
+      payload: new Error(errorMsg),
+      scopes: [],
+    });
+
+    expect(errorSpy).toHaveBeenCalledTimes(1);
+    const args = errorSpy.mock.calls[0];
+
+    expect(consoleCallInludes(args, errorMsg)).toBe(true);
```

Comment:
I'm not sure if this is a big deal but I came here looking to see how things should be outputted, since I found the code for compiling the messages a bit confusing, and not being able to look anywhere and see examples of the output makes that somewhat difficult.

Other than having fixture files or using snapshots I'm not sure of a good way to address that though.

Reply:
Yeah, I didn't want to tie the tests to the formatting too much, especially since we use chalk, so we can't perform strict equality checks on just the content without duplicating a lot of the logic. I was trying to sidestep that by checking that pieces of the output are included, since we don't care much about the colors, etc. in this alpha release.

For example, we want the message to be there, but if we later decide to move the `scopes` to after the text, or change the concept of `scopes` to `tags`, I didn't want the tests to be too brittle.

I think that testing the formatting more strictly makes a lot of sense after the 1.0 release, though, where code changes should be purely architectural. Perhaps snapshot testing would be good at that point?